File: app/src/main/java/ru/prsolution/winstrike/mvp/common/PrefUtils.java
package ru.prsolution.winstrike.mvp.common;

import android.content.Context;
import android.content.SharedPreferences;

import ru.prsolution.winstrike.WinstrikeApp;


/**
 * Date: 18.01.2016
 * Time: 15:01
 *
 * @author Yuri Shmakov
 */
public class PrefUtils {
	private static final String PREF_NAME = "github";

	public static SharedPreferences getPrefs() {
		return WinstrikeApp.getInstance().getAppComponent().getContext().getSharedPreferences(PREF_NAME, Context.MODE_PRIVATE);
	}

	public static SharedPreferences.Editor getEditor() {
		return getPrefs().edit();
	}
}

File: app/src/main/java/ru/prsolution/winstrike/mvp/common/PrefUtils.kt
package ru.prsolution.winstrike.mvp.common

import android.content.Context
import android.content.SharedPreferences

import ru.prsolution.winstrike.WinstrikeApp


/**
 * Date: 18.01.2016
 * Time: 15:01
 *
 */
object PrefUtils {
    private val PREF_NAME = "token"

    val prefs: SharedPreferences
        get() = WinstrikeApp.getInstance().appComponent.context.getSharedPreferences(PREF_NAME, Context.MODE_PRIVATE)

    val editor: SharedPreferences.Editor
        get() = prefs.edit()
}
